pkg/volume/gce_pd/gce_util.go

```diff
@@ -101,52 +105,95 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 		case "zones":
 			zonesPresent = true
 			configuredZones = v
+		case "replica-zones":
+			replicaZonesPresent = true
+			configuredReplicaZones = v
 		case volume.VolumeParameterFSType:
 			fstype = v
 		default:
 			return "", 0, nil, "", fmt.Errorf("invalid option %q for volume plugin %s", k, c.plugin.GetPluginName())
 		}
 	}
 
-	if zonePresent && zonesPresent {
+	if zonePresent && zonesPresent && replicaZonesPresent {
```

Comment:
You could do (zonePresent || zonesPresent) && replicaZonesPresent to handle all 3 cases.

Reply:
Done. Gets rid of the custom error message per condition, but no big deal.